runtime/databricks/automl_runtime/forecast/pmdarima/training.py

```diff
@@ -141,8 +141,9 @@ def _fill_missing_time_steps(df: pd.DataFrame, frequency: str):
     @staticmethod
     def _validate_ds_freq(df: pd.DataFrame, frequency: str):
```

Comment:
I'm actually pretty surprised that we don't need to cast the ds manually when feeding it into ARIMA. Can we add a test for the AriamEstimator fitting with monthly timeseries to make sure it actually fits?